packages/slate-react/src/utils/scroll-to-selection.js

```diff
@@ -15,6 +15,12 @@ const OVERFLOWS = [
   'scroll',
 ]
 
+/**
+ * Detect whether we are running IOS version 11
+ */
+
+const IS_IOS_11 = IS_IOS && !!window.navigator.userAgent.match(/os 11_/i)
```

Comment:
This will throw in server-side rendering I think. Maybe we can just put the `11` check inside the `scrollToSelection` function instead?

Reply:
Mentioned this in our Slack conversation. It shouldn't throw an error because of the `IS_IOS` check before the window userAgent check. `IS_IOS` will only return true if we are in a browser. That check is done in the `constants/environment.js` file.

Because the check was in IS_IOS already, I took out the check here.

Reply:
Ah gotcha, that makes sense. It's slightly confusing maybe, but I think it sounds good. Please ignore me!

Reply:
Yeah, I agree that it is confusing. If this was my own code, I usually put comments right next to it but I noticed that SlateJS code is pretty sparse with comments so I tried to follow that commenting style.